components/datetime/src/provider/time_zones.rs

```diff
@@ -421,12 +421,22 @@ pub(crate) mod legacy {
             );
 
             assert_eq!(
-                converted.get().get(tz, t).unwrap().1.map(|mz| mz.id),
+                converted
+                    .get()
+                    .get(tz, t)
+                    .unwrap()
+                    .1
+                    .map(|mz| match mz.id.get() {
+                        // the ID list changed with CLDR 48
+                        22 => 21,
+                        31 => 30,
+                        _ => unreachable!(),
+                    }),
```

Comment:
Issue: This looks bad; we have checksums to keep the lists in sync. Why do you need this?

Reply:
I guess we can delete the test because the checksum doesn't match anymore

Reply:
Don't delete tests! `test_metazone_timezone_compat` is still a useful test

Reply:
what's your issue then